app/controllers/api/v1/ndc_sdgs_controller.rb

```diff
@@ -18,11 +18,11 @@ def show
       # Temporary hack to render NDC-SDG linkages for countries
       # that do not have full NDCs imported
       def show_by_location
-        @location = Location
-          .includes(:ndc_targets)
-          .where(iso_code3: params[:code].upcase)
-          .first
-        unless @location.ndc_targets.length > 0
+        @location = Location.
+          includes(:ndc_targets).
+          where(iso_code3: params[:code].upcase).
+          first
+        if @location.ndc_targets.length.empty?
```

Comment:
You made Rubocop happy, but made me sad :P `undefined method empty? for Integer` . You forgot to remove the `length` bit. I'll fix and merge.